### DISCUSSION TITLE
Bulk configuration for "Track inventory"

### DISCUSSION THREAD

**Discussion by falko-burghausen** [saleor/saleor#13322] [Q&A]
Hi, is there a way to bulk-edit the track inventory checkbox within one product for all variants? ATM this is quite time-consuming to disable it manually for lots of variants and lots of products (as it also cannot be done in the table view for the variants). Or is there a way to configure Saleor to disable inventory tracking by default?

Thanks for your help!

<img width="1460" alt="Bildschirmfoto 2023-07-05 um 13 44 11" src="https://github.com/saleor/saleor/assets/22614664/05e62c01-b9fb-40ba-a29e-01fac241fe03">

**Comment by timuric** [comment]
hi @falko-burghausen this an interesting idea, we would consider that. For now a workaround could be to setup monitoring of your configuration preferences, for example you could use BI tool like metabase that can be connected to your saleor database to report on misconfigurations so that the admins get an email when they forget to uncheck the box for tracking. This can be useful also for other merchandising standards like filling in the seo fields
